sheets.js

```diff
@@ -111,9 +111,12 @@ function createRows(destSheet, data) {
   try{
     let columnNames = getSheetHeaderNames(destSheet)
     let values = data.map(row => {
-    return columnNames.map(colName => row[colName] ? row[colName] : null)
+      return columnNames.map(colName => row[colName] ? row[colName] : null)
     })
-    destSheet.getRange(destSheet.getLastRow()+1, 1, values.length, values[0].length).setValues(values)
+    let newRows = destSheet.getRange(destSheet.getLastRow()+1, 1, values.length, values[0].length)
+    newRows.setValues(values)
+    //fixRowDataValidation(newRows)
+    //fixRowNumberFormatting(newRows)
```

Comment:
Did you mean to leave these lines commented out?

Reply:
Yes. I wanted to leave the reference to them there, as a reminder that the method for validation/formatting is outdated, but they really slow things down here. 
For now, the recommended method is the function that checks validation and formatting for the whole sheet -- tests showed that these functions still only fix a minor number of issues but add a significant overhead.

Reply:
OK, thanks. So sheet-level data validation and formatting would then happen outside of this process entirely? As in, when the user sees a problem or as a scheduled task?

Reply:
Yeah, I'd like to see how that goes for now. I have a suspicion that these lines were not helping particularly much, but also I will just leave them commented out in case we realize they were really helping, and want to turn them back on. They have sort of limited ability to fix things, so I don't know if they are worth the cost.

Reply:
OK, thanks for explaining. I'll merge this in.